src/Controller/UI/StatsController.php

```diff
@@ -0,0 +1,299 @@
+<?php
+
+namespace App\Controller\UI;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Query;
+
+use Pelagos\Bundle\AppBundle\Twig\Extensions as TwigExtentions;
```

Comment:
Need to change namespace for this

Reply:
good catch. I wonder how it's even working, or if it's even needed.

Reply:
If you goto the /stats/json route, it breaks and that is used in the drupal home page